templates/rally-beyond-the-paywall/pioneer-enrollment/pioneer-enrollment.1.schema.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Please configure this ping like this: https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/main/templates/pioneer-citp-news-disinfo-two/pioneer-enrollment/pioneer-enrollment.1.schema.json